DEVELOPMENT.md

```diff
@@ -230,13 +230,15 @@ We run integration tests by running it in Cinema 4D's python i.e. c4dpy for subm
          test_cinema4d.py       # Runs all the tests scenes in a parametrized fashion.
 </pre>
 
-You would have to setup Cinema 4D licensing before you run the tests.
+You would have to setup Cinema 4D and Redshift licensing before you run the tests.
+For Cinema 4D licensing, set environment variable by using `$env:g_licenseServerURL = <your-license-server-host>:<port>` on Windows Powershell. 
+For redshift licensing, set the environment variable by using `$env:redshift_LICENSE = <port>@<your-license-server-host>` on Windows Powershell. 
 
 1. Set the environment variable `C4D_PYTHON` to the installation folder of Cinema 4D.
    1. `set C4D_PYTHON=<Cinema 4D location>` on Windows Command or `$env:C4D_PYTHON = <Cinema 4D location>` on Windows Powershell.
       1. The default location for `Cinema 4D` on Windows is `C:\Program Files\Maxon Cinema 4D 2025\`. This location would be automatically used if the directory exists.
 2. For running the adaptor tests, we would need to install `pywin32` to the installation paths as its an adaptor dependency.
    2.1 Run `pip install pywin32==308 -t <your Python site-packages location>`. 
-   During my testing, pywin32's version 308 was required because of other dependencies requiring this version.s
+   During my testing, pywin32's version 308 was required because of other dependencies requiring this version.
```

Comment:
This looks hacky. Do we need to fix our dependencies or something like that? Out of scope of this PR.